bundles/org.openhab.binding.lifx/src/main/java/org/openhab/binding/lifx/internal/LifxLightDiscovery.java

```diff
@@ -199,6 +199,7 @@ protected void doScan() {
             }
         } catch (Exception e) {
             logger.debug("{} while discovering LIFX lights : {}", e.getClass().getSimpleName(), e.getMessage());
+            isScanning = false;
```

Comment:
LGTM, but why do we catch `Exception` here?

Reply:
After checking all thrown exceptions, it has been changed to just catching an IOException.